app/controllers/review_controller.rb

```diff
@@ -0,0 +1,58 @@
+class ReviewController < ApplicationController
+
+  def index
+    @reviews = Review.all
+  end
+
+  def show
+    @review = Review.find(params[:id])
+  end
+
+  def edit
+    @review = Review.find(params[:id])
+  end
+
+  def update
+    @review = Review.find(params[:id])
+    if @review.update_attributes(review_params)
+      #redirect and flash message
+    else
+      #render and flash message
```

Comment:
You should delete these comments, and either fully flesh out the functionality or save this action for a later branch!